#modified calculator Using Exception.
class Calculator:

    def __init__(self, a, b): 
        self.a = a
        self.b = b

    def add(self):
        return self.a + self.b
    
    def sub(self):
        return self.a - self.b
    
    def mul(self):
        return self.a * self.b
    
    def div(self):
        try:
            return self.a / self.b
        except ZeroDivisionError:
            return "Cannot divide by zero"
    
    def mod(self):
        try:
            return self.a % self.b
        except ZeroDivisionError:
            return "Cannot divide by zero"

calc1 = Calculator(10, 0)
print(calc1.add())
print(calc1.sub())
print(calc1.mul())
print(calc1.div())
print(calc1.mod())
